docs/details/shards/GFX/glTF.md

```diff
@@ -0,0 +1,21 @@
+- This shard can take a table as input that contains the following keys with the following values:
+  - "transform": The 4x4 transformation matrix that contains the translation, rotation and scale for the glTF model to adopt.
+  - "materials": A sub-table that contains information to modify the materials of the glTF model.
+  - "nodes" (a sequence of strings representing the nodes to modify): A sub-table that contains information to modify the nodes of the glTF model.
```

Comment:
Clarify that it's not actually a key called "nodes"
instead a node can be customized be passing the path as a sequence, like:
```dart
{
  ["rootNode" "spine" "upperBody" "etc."]: {
    scale: @f3(2.0)
  }
} GFX.glTF(...)
```